Decred Wallet/Features/Navigation Menu/NavigationMenuViewController.swift

```diff
@@ -157,6 +157,13 @@ class NavigationMenuViewController: UIViewController {
         self.slideMenuController()?.changeMainViewController(self.currentMenuItem.viewController, close: true)
         self.navMenuTableView.reloadData()
     }
+    
+    func invalidateSyncTimer() {
```

Comment:
What is `syncTimer` that is being invalidated? Does sync use a timer? I think `stopRefreshingBestBlockAge` is a more descriptive function name.

Reply:
Renamed function